src/Microcosm.js

```diff
@@ -1,18 +1,17 @@
+import Debug       from './plugins/debug'
 import Diode       from 'diode'
+import History     from './plugins/history'
 import MetaStore   from './stores/meta'
+import State       from './plugins/state'
 import Transaction from './Transaction'
-import Tree        from './Tree'
-import coroutine   from './coroutine'
 import defaults    from './defaults'
-import dispatch    from './dispatch'
-import eventually  from './eventually'
 import flatten     from './flatten'
 import install     from './install'
 import lifecycle   from './lifecycle'
 import merge       from './merge'
 import tag         from './tag'
 
-let Microcosm = function() {
+function Microcosm (options) {
```

Comment:
Specifically, for constructors! I totally want the name Microcosm to show up in the console.